### PR TITLE
Ajax example

### PR DESCRIPTION
**Pull request by Martin-Luther**
### 4.1.3

 * Martin-Luther ETOUMAN:
   - Add jQuery Ajax example: `ajax-upload`.   
     `$ node ajax-upload`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules/
 coverage/
 npm-debug.log
+.DS_Store**
```
Comment (andrewrk) [inline .gitignore]: this goes in your user scoped git ignore file, not the project scoped one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+### 4.1.3
```
Comment (andrewrk) [inline CHANGELOG.md]: leave it up to the maintainers to update the changelog.

**Comment by andrewrk** [comment]
I appreciate that you took the time to make a contribution, but this doesn't seem to bring anything new to the table in terms of example code demonstrating how to use multiparty. We certainly don't need to demonstrate how to use jQuery.

**Comment by Martin-Luther** [comment]
Producing code without giving precious informations on how to use it is useless.
The code sample illustrates how to upload files using XMLHttpRequest along with your project: from frontend to backend. May be, you are so deep in your stuff that you don't even see the value in what I have pushed.
I have been searching for 3 days how to upload files using ajax (with or without jquery). I have tested 3 other crappy modules before trying multiparty ... If I had seen this code sample in your examples, It would have saved me a lot of time. Isn't contributing about sharing experience and saving time ?

File: CHANGELOG.md (hunk 2 above)
Comment (Martin-Luther) [inline CHANGELOG.md]: I did not now .. no documentation on how to contribute .. sorry.

**Comment by andrewrk** [comment]
I'm sorry that it took so long for you to get it working, but I still don't think the example belongs in this codebase. Maybe you could maintain a repository called "node-uploading-examples" or something.

The way that you use multiparty is the same as this example: https://github.com/andrewrk/node-multiparty/blob/master/examples/upload.js

The only difference is that you use jQuery. I don't think this node-multiparty repository needs an example of how to use jQuery. If anything, I would want to discourage the use of that library, but that's not really relevant to multiparty.

File: CHANGELOG.md (hunk 2 above)
Comment (andrewrk) [inline CHANGELOG.md]: it's okay, I'm just giving you (hopefully constructive) feedback.

**Pull request closed without merging**